### ISSUE TITLE
Async method 'asyncData' has no 'await' expression

### ISSUE THREAD

**Issue by iBobik** [nuxt/eslint-config#87]
When [official examples](https://nuxtjs.org/guide/async-data) recommends something like this:

```
export default {
  async asyncData ({ params }) {
    const { data } = await axios.get(`https://my-api/posts/${params.id}`)
    return { title: data.title }
  }
}
```

Then official eslint config should not report it as error.

**Comment by manniL** [comment]
The error message

>  Async method 'asyncData' has no 'await' expression

Doesn't fit to the code, as the async method `asyncData` has an await expression (`await axios...`) though 🤔

**Comment by iBobik** [comment]
Sorry, copied the wrong example :-)

```
export default {
  asyncData ({ params }) {
    return axios.get(`https://my-api/posts/${params.id}`)
      .then((res) => {
        return { title: res.data.title }
      })
  }
}
```

**Comment by manniL** [comment]
Would happily accept a PR! 😋	https://github.com/nuxt/docs/blob/master/en/guide/async-data.md
